src/sourabh/BankingSystem.java

```diff
@@ -0,0 +1,60 @@
+package sourabh;
+
+public class BankingSystem {
+	String custName;
+	int totalAmount;
+	int debitCount=0; 
```

Comment:
Not require to give 0 value to variable, if you are not initialised any variable with any value then it will initialised with it's default value. and this balance should not hard coded.